winui/Kanban/columns.md

```diff
@@ -386,6 +386,8 @@ public class ViewModel
 {% endhighlight %}
 {% endtabs %}
 
+![column-mappingpath-in-winui-kanban](images/columns/column-mappingpath-in-winui-kanban.png)
```

Comment:
Rename to `mapping-path`

Reply:
Renamed and merged changes in this [PR](https://github.com/syncfusion-content/winui-docs/pull/1047).